backend/data/processors/manual_fixes/COMMFixes.py

```diff
@@ -27,38 +27,38 @@
 
 def fix_conditions():
     """ Functions to apply manual fixes """
-
+    
     CONDITIONS["COMM0999"][PROCESSED] = COMM_0999()
-    CONDITIONS["COMM1040"] = COMM_1040(CONDITIONS["COMM1040"])
+    #CONDITIONS["COMM1040"] = COMM_1040(CONDITIONS["COMM1040"])
     CONDITIONS["COMM1100"] = COMM_1100(CONDITIONS["COMM1100"])
     CONDITIONS["COMM1110"] = COMM_1110_1120_1140(CONDITIONS["COMM1110"])
-    CONDITIONS["COMM1120"] = COMM_1110_1120_1140(CONDITIONS["COMM1120"])
-    CONDITIONS["COMM1140"] = COMM_1110_1120_1140(CONDITIONS["COMM1140"])
+    #CONDITIONS["COMM1120"] = COMM_1110_1120_1140(CONDITIONS["COMM1120"])
+    #CONDITIONS["COMM1140"] = COMM_1110_1120_1140(CONDITIONS["COMM1140"])
     CONDITIONS["COMM1150"] = COMM_1150(CONDITIONS["COMM1150"])
-    CONDITIONS["COMM1170"] = COMM_1170_1180(CONDITIONS["COMM1170"])
-    CONDITIONS["COMM1180"] = COMM_1170_1180(CONDITIONS["COMM1180"])
-    CONDITIONS["COMM1190"] = COMM_1190(CONDITIONS["COMM1190"])
+    #CONDITIONS["COMM1170"] = COMM_1170_1180(CONDITIONS["COMM1170"])
```

Comment:
feel free to completely delete these lines :)

Reply:
as well as their corresponding functions